### DISCUSSION TITLE
RocketMQ traceId is not printed

### DISCUSSION THREAD

**Discussion by muarine** [apache/skywalking#11074] [Q&A]
using maven dependency

> <dependency>
    <groupId>com.alibaba.cloud</groupId>
    <artifactId>spring-cloud-starter-stream-rocketmq</artifactId>
    <version>2.2.8.RELEASE</version>
  </dependency>

I encountered the problem that traceId does not print. The specific consumption configuration and code are as follows:

> spring:
  cloud:
    function:
      definition: archive;
    stream:
      rocketmq:
        binder:
          name-server: ${rocketmq.producer.nameServer}
          access-key: ${rocketmq.producer.accessKey}
          secret-key: ${rocketmq.producer.secretKey}
          vip-channel-enabled: true
        bindings:
          archive-in-0:
            consumer:
              subscription: '${rocketmq.consumer.tag}'
      bindings:
        archive-in-0: 
          destination: ${rocketmq.consumer.topic}  
          content-type: application/json
          group: ${rocketmq.consumer.group}
          binder: rocketmq
      binders:
        rocketmq:
          type: rocketmq
      default-binder: rocketmq

> @Bean
    @Qualifier("archive")
    public Consumer<Message<String>> archive() {
        log.info("archive consumer start ..");
        return new Consumer<Message<String>>() {
            @Override
            public void accept(Message<String> msg) {
                try {
                    String payload = msg.getPayload();
                    log.info("Consumer :{}", payload);
                    Archive archive = JacksonHelper.toObj(payload, Archive.class);
                    if (ObjectUtils.isNotEmpty(archive)) {
                        ArchiveConsumer.this.doConsumerHandle(archive);
                    }
                } catch (Exception e) {
                    log.warn("archive-Consumer error:{}", e.getMessage());
                }
            }
        };
    }

result:
>[2023-07-11 14:23:55:660] [ConsumeMessageThread_GID_xxxx_dev_1] [N/A] [INFO] [com.xxxx.archive.ArchiveConsumer] ->>   archive-Consumer :{"archivesInfoId":"1678651312592064500","operationTypename":"new data","operationTypeCode":"add","operationTime":"2023-07-11 14:23:55"}

I have also viewed the Demo code and Skywalking plugin rocketmq4.x in showcase, started the agent and started online debugging of service consumption messages. rocketmq4.x plugin is not active (orderly and concurrent),
![image](https://github.com/apache/skywalking/assets/3285738/9ceb0efc-78c9-411c-b496-2c9e6f6d67cf)
> queto: 
com.alibaba.cloud.stream.binder.rocketmq.integration.inbound.RocketMQInboundChannelAdapter#onInit
com.alibaba.cloud.stream.binder.rocketmq.integration.inbound.RocketMQInboundChannelAdapter#consumeMessage

Looking forward to your reply, thanks!

**Comment by wu-sheng** [comment]
Quite similar case as  https://github.com/apache/skywalking/discussions/11043 

We don't know much about a specific plugin, basically. You have to check the source codes or reach RocketMQ upstream community

**Reply by muarine** [reply]
thanks, I will take a look
